libfwupdplugin/fu-udev-device.h

```diff
@@ -118,3 +118,5 @@ gboolean	 fu_udev_device_write_sysfs		(FuUdevDevice	*self,
 							 GError		**error)
 							 G_GNUC_WARN_UNUSED_RESULT;
 const gchar	*fu_udev_device_get_devtype		(FuUdevDevice	*self);
+GPtrArray 	*fu_udev_device_get_siblings_with_subsystem (FuUdevDevice *self,
```

Comment:
siblings or children? I think we use the latter in other places, but I don't care either way too much.

Reply:
This is specifically finding siblings by looking only for devices with the same parent as self. Getting children with a given subsystem would be a different operation.

Reply:
Gah, of course. Sorry for the noise.